synapse/storage/schema/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# When updating this value, please leave a short summary of the changes below.
```

Comment:
Hmm, should we be asking the same for `SCHEMA_COMPAT_VERSION` perhaps?

Reply:
I wouldn't really think so. it's somewhat obvious?

Reply:
The same could be said about this value :stuck_out_tongue:

I might just tweak the wording slightly to apply to both values.